java-frontend/src/main/java/org/sonar/java/se/constraint/Constraint.java

```diff
@@ -22,4 +22,11 @@
 public interface Constraint {
 
   boolean isNull();
+
+  /**
+   * @return String representation of value encoded by constraint for purpose of flow message
+   */
+  default String valueAsString() {
+    return "";
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this method and declare a constant for this value. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS3400)